src/lib/simulations/statSimulationController.tsx

```diff
@@ -58,17 +58,17 @@ export function saveStatSimulationBuildFromForm() {
   // Check for invalid button presses
   if (simType == StatSimTypes.Disabled || !form.statSim?.[simType]) {
     console.warn('Invalid sim', form, simType)
-    return
+    return -1
   }
 
   // Check for missing fields
   const simRequest: SimulationRequest = form.statSim[simType]
   if (!validateRequest(simRequest)) {
     console.warn('Invalid sim', form, simType)
-    return
+    return -1
   }
 
-  saveStatSimulationRequest(simRequest, simType, true)
+  return saveStatSimulationRequest(simRequest, simType, true)
```

Comment:
Not a big fan of returning numbers from things that shouldn't be numbers, I think we should keep the return type as null where it fails